src/sentry/incidents/endpoints/serializers.py

```diff
@@ -75,15 +75,25 @@ class AlertRuleTriggerActionSerializer(CamelSnakeModelSerializer):
     id = serializers.IntegerField(required=False)
     type = serializers.CharField()
     target_type = serializers.CharField()
+    use_async_lookup = serializers.BooleanField(required=False, default=False)
 
     class Meta:
         model = AlertRuleTriggerAction
-        fields = ["id", "type", "target_type", "target_identifier", "integration", "sentry_app"]
+        fields = [
+            "id",
+            "type",
+            "target_type",
+            "target_identifier",
+            "integration",
+            "sentry_app",
+            "use_async_lookup",
+        ]
         extra_kwargs = {
             "target_identifier": {"required": True},
             "target_display": {"required": False},
             "integration": {"required": False, "allow_null": True},
             "sentry_app": {"required": False, "allow_null": True},
+            "use_async_lookup": {"required": False, "allow_null": True},
```

Comment:
You shouldn't need this since you're declaring `use_async_lookup` directly rather than inferring via the model params.